uitools/examples/cpp_quick/src/demos/ScalebarDemo.h

```diff
@@ -0,0 +1,28 @@
+// Copyright 2022 ESRI
+//
+// All rights reserved under the copyright laws of the United States
+// and applicable international laws, treaties, and conventions.
+//
+// You may freely redistribute and use this sample code, with or
+// without modification, provided you include the original copyright
+// notice and use restrictions.
+//
+// See the Sample code usage restrictions document for further information.
+#ifndef ARCGIS_RUNTIME_TOOLKIT_CPP_QUICK_DEMO_SCALEBARDEMO_H
+#define ARCGIS_RUNTIME_TOOLKIT_CPP_QUICK_DEMO_SCALEBARDEMO_H
+
+#include "BaseDemo.h"
+
+#include <QObject>
+#include <QQmlEngine>
+
+class ScalebarDemo : public BaseDemo
```

Comment:
Out of curiosity, where are the map/scene set? Shouldn't they be set here via the protected `initMap_` and `initScene_` ?

Reply:
Sounds good, thank you for the explanation!